### ISSUE TITLE
Drop down options are not read by the Zoomtext speech option

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#822]
Zoomtext speech option only reads out the selected item on the drop down menu and doesn't read the other available items on the drop down list. This was tested on all the drop down menus for creating an FRO page and a project and we had the same problem on all drop down menus.

**Comment by greatislander** [comment]
@sepidehshahi — can we get more context?
